readme.txt

```diff
@@ -19,7 +19,7 @@ Newspack Newsletters comes with four default layouts to help you start building
 
 = Use Gutenberg blocks and settings =
 
-Newspack Newsletters supports most core Gutenberg blocks and configurations, letting you use blocks like Group, Button, Columns, and more, to create dynamic layouts. The plugin will warn you when a block will not display as expected in an email. Newspack Newsletters uses [MJML](https://mjml.io/) to transform the HTML generated by Gutenberg into markup that will be displayed nicely by email clients.
+Newspack Newsletters supports most core Gutenberg blocks and configurations, letting you use blocks like Group, Button, Columns, and more, to create dynamic layouts. The plugin will warn you when a block will not display as expected in an email.
```

Comment:
IMO this information was necessary because the plugin asked users to get MJML API credentials. With that not needed anymore, this information is an implementation detail irrelevant for non-technical users.

Reply:
I can see that argument. I wouldn't say it's totally irrelevant, though.